app/models/comment.rb

```diff
@@ -1,4 +1,15 @@
+require 'csv'
+
 class Comment < ApplicationRecord
   include Visible
   belongs_to :post
+
+  def self.to_csv
+    CSV.generate do |csv|
+      csv << ["Post title", "Comment author", "Comment body"]
+      all.each do |comment|
```

Comment:
Does `all` reference what this method was called on or does it call a method getting everything? It seems to be the latter, but can you please explain to me how it works?